src/components/Event/Event.js

```diff
@@ -1,20 +1,14 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { Map } from 'immutable';
-import timeAgo from 'damals';
 
 import { getReadersMap } from '../../redux/modules/readers';
 import { getTagsMap } from '../../redux/modules/tags';
 import { getUsersMap } from '../../redux/modules/users';
 
-import classes from './Event.css';
-
-const SECOND = 1e3;
-const MINUTE = 60 * SECOND;
-const HOUR = 60 * MINUTE;
+import { TimeAgo } from '../TimeAgo/TimeAgo.js';
 
-const ELAPSED_MIN = MINUTE;
-const ELAPSED_MAX = HOUR;
+import classes from './Event.css';
```

Comment:
lol wut?!  thats pretty cool!

I take it this is enabled via a babel plug in? or is it built in?